src/integrationTest/resources/application-itest.yaml

```diff
@@ -0,0 +1,21 @@
+idam:
+  api:
+    url: http://localhost:${wiremock.server.port:5000}
```

Comment:
Wouldn't this be

`url: http://localhost:${wiremock.server.port}`

so without the default value - otherwise isn't there a risk that integration tests could run against local docker instances rather than wiremock ?

Reply:
same for all values below